src/charm.py

```
@@ -997,22 +997,27 @@ def _execute_anonymize_cmd(self, event: ActionEvent) -> Iterator[str]:
                 email,
             ]
 
-            if container.can_connect():
-                process = container.exec(
-                    cmd,
-                    user="indico",
-                    working_dir="/srv/indico",
-                    environment=indico_env_config,
+            if not container.can_connect():
+                logger.error(
```

Can you set a failed status for the action with the appropriate message instead of just logging the error?